blockchain/indexbuilder.go

```diff
@@ -181,6 +181,13 @@ func (ib *IndexBuilder) initAndLoadActions() error {
 		if err != nil {
 			return err
 		}
+	} else {
+		if err = ib.dao.deleteBucket(blockAddressActionMappingNS); err != nil {
```

Comment:
So if we want to reindex, we need to cleanup existing addr <-> act mapping?

Why don't we need to cleanup other index data?

Reply:
The index of all ations will be covered when reindex